tools/winmd2objc/include/CodeGen.h

```diff
@@ -0,0 +1,92 @@
+//******************************************************************************
+//
+// Copyright (c) Microsoft. All rights reserved.
+//
+// This code is licensed under the MIT License (MIT).
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+//
+//******************************************************************************
+
+#pragma once
+
+#include <iostream>
+#include <vector>
+#include <map>
+#include <memory>
+#include "ObjectModel.h"
+#include "CommandLine.h"
+
+const std::wstring g_defaultOutputDirectory = L"..\\..\\..\\Projections\\";
```

Comment:
Yes i do override it with -output.

---
In reply to: [117095042](https://github.com/Microsoft/WinObjC/pull/2708#discussion_r117095042) [](ancestors = 117095042)